dipy/tracking/tests/test_fbc.py

```diff
@@ -0,0 +1,49 @@
+from dipy.denoise.enhancement_kernel import EnhancementKernel
```

Comment:
We do not have automated coverage for cython files. Can you make a list of the functions that are being tested by this test? We need to make sure that all functions are tested.